application-xpoll-api/src/main/java/com/xwiki/xpoll/rest/XPollResource.java

```diff
@@ -47,7 +47,7 @@
      * @return returns 404 if the page doesn't exist or doesn't have a XPollClass object, 200 otherwise
      * @throws XWikiRestException when failing to find the document or the document is missing the poll
      */
-    @PUT
+    @POST
```

Comment:
Please rollback. ``PUT`` is the right method here. See https://restfulapi.net/rest-put-vs-post/ .